deploy/osps/default/osp-amzn2.yaml

```diff
@@ -415,6 +415,13 @@ spec:
               kernel.panic_on_oops = 1
               kernel.panic = 10
               net.ipv4.ip_forward = 1
+              {{- if or (eq .NetworkIPFamily "IPv4+IPv6") (eq .NetworkIPFamily "IPv6+IPv4") (eq .NetworkIPFamily "IPv6") }}
+              net.ipv6.conf.all.forwarding = 1
+              # Configure Linux to accept router advertisements to ensure the default
+              # IPv6 route is not removed from the routing table when the Docker service starts.
+              # For more information: https://github.com/docker/for-linux/issues/844
+              net.ipv6.conf.all.accept_ra		= 2
```

Comment:
@moadqassem we had a similar fix for flatcar through https://github.com/kubermatic/operating-system-manager/pull/198 but i'm wondering if we really need this. Maybe this should be up to the user to configure in case route advertisement fails. Since this is something that'd be governed by the upstream network so eventually the cloud provider.

Reply:
I agree, the user should decide how the traffic is forward across different cloud provider. I mean in general we can include this in an if statement

Reply:
In my opinion it makes not really sense for Kubernetes workers to not accept RA.